packages/ref-doc/src/emitters/docusaurus.ts

```diff
@@ -63,30 +63,42 @@ function renderIndexFile(refDoc: TypeSpecRefDoc): string {
     content.push(headings.h2(namespace.id), "");
     if (namespace.decorators.length > 0) {
       content.push(headings.h3("Decorators"), "");
+      const listContent = [];
       for (const decorator of namespace.decorators) {
-        content.push(` - [${inlinecode(decorator.name)}](./decorators.md#${decorator.id})`);
+        listContent.push(` - [${inlinecode(decorator.name)}](./decorators.md#${decorator.id})`);
+        listContent.sort();
```

Comment:
Sort maybe only at the end of the for instead of each time we adding?

Reply:
You have to sort each list independently. You can't just sort at the end.

Reply:
I just mean sort it right before you … he lost content into the main content instead of each time you push to it

Reply:
Ah yes, now I see. Thank you!